src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -14,7 +14,7 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
   </metadata>
   <files>
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
+    <file src="Microsoft.TestPlatform.Build\**\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
```

Comment:
Yeah on windows it goes to artifacts\Debug\Microsoft.TestPlatform.Build\netstandard1.3, on *nix it goes to artifacts/Debug/Microsoft.TestPlatform.Build/Debug/netstandard1.3/ . I can investigate why this happens instead.